bindings_ffi/src/lib.rs

```diff
@@ -65,8 +70,21 @@ pub async fn create_client(
         .await
         .map_err(|e| stringify_error_chain(&e))?;
 
+    let store = match db {
+        Some(path) => {
+            info!("Using persistent storage: {} ", path);
+            EncryptedMessageStore::new_unencrypted(StorageOption::Persistent(path))
```

Comment:
Shouldn't we also take in an encryption key here from the caller, and provide an encrypted DB by default?

Reply:
Yep you're totally right. Added the ability to pass a key in https://github.com/xmtp/libxmtp/pull/219/commits/54bcfeaaa7bac73db12d9aa33ac8feea33e9e579. We can ensure clients _always_ pass a key in the SDKs.